tokio/src/time/mod.rs

```diff
@@ -54,12 +54,12 @@
 //! # }
 //! ```
 //!
-//! A simple example using [`interval`] to execute a task every two seconds.
+//! A simple example using `interval` to execute a task every two seconds.
 //!
-//! The difference between [`interval`] and [`sleep`] is that an
-//! [`interval`] measures the time since the last tick, which means that
-//! `.tick().await` may wait for a shorter time than the duration specified
-//! for the interval if some time has passed between calls to `.tick().await`.
+//! The difference between `interval` and [`sleep`] is that an `interval`
+//! measures the time since the last tick, which means that `.tick().await`
+//! may wait for a shorter time than the duration specified for the interval
+//! if some time has passed between calls to `.tick().await`.
```

Comment:
There's no need to remove the links to `interval` here.

Reply:
fixed